src/main/java/uk/gov/companieshouse/api/accounts/configuration/AmazonS3Configuration.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.companieshouse.api.accounts.configuration;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+
+@Configuration
+public class AmazonS3Configuration {
+
+    @Autowired
+    private EnvironmentReader environmentReader;
+
+    @Bean
+    public AmazonS3 getAmazonS3() {
+        return AmazonS3ClientBuilder
+            .standard()
+            .withRegion(Regions.EU_WEST_1)
```

Comment:
We shouldn't hardcode this - Steve Bowen is currently working on moving everything to eu west 2 (London), so it should be configurable

Reply:
Good point. I will change it.